src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { TextField } from 'react-md';
+import StellarValueRange from '../../../charts/shared/StellarValueRange';
+
+class FillableTableRangeInput extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    const { answer } = props;
+
+    this.state = {
+      minValue: answer.data[0] || undefined,
+      maxValue: answer.data[1] || undefined,
```

Comment:
It looks like this is barfing when `answer` is `null` (or maybe it's `undefined` or whatever).  You can probably get around this by using the "Optional chaining" operator `?.`, so like `answer?.data?.[0]` which should then try it, and if fails land in the other side of your `||` (so set the value as `undefined`).  

My guess is this is only happening when the user has never answered this question, but it could be the case that the `answer` prop never has value in the constructor.  If so, let me know and we can brainstorm how/when to peek the value in `componentDidMount` or `componentDidUpdate`.